lesson01.calculator/index.html

```diff
@@ -1,22 +1,318 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <title>Web Calculator</title>
-    <meta charset="utf-8">
-    <meta name="viewport"
-          content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <style>
-        body {
+  <title>Web Calculator</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <style>
+    body {
+      font-family: arial, sans-serif;
+    }
 
-        }
-    </style>
+    .calc {
+      display: grid;
+      grid: repeat(6, 1fr) / repeat(4, 1fr);
+      grid-gap: 5px;
+      box-sizing: border-box;
+      padding: 5px;
+      height: 95vh;
```

Comment:
95vh?
Maybe you should use a containing element with padding (you can use `<body?`), and give it 100vh